### PR TITLE
Debt details API

### PR DESCRIPTION
**Pull request by lizhengwei88**
Debt details API

### CODE REVIEW

File: api/handlers/blockinfo_handler.go [hunk 1]
```diff
@@ -790,6 +790,30 @@ func (h *BlockHandler) GetTxs() gin.HandlerFunc {
 	}
 }
 
+//GetDebtByHash handler for get debt by hash
+func (h *BlockHandler) GetDebtByHash() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		dbClinet := h.DBClient
```
Comment (WTer) [inline api/handlers/blockinfo_handler.go]: dbClinet should be dbClient

File: api/handlers/blockinfo_handler.go [hunk 2]
```diff
@@ -790,6 +790,30 @@ func (h *BlockHandler) GetTxs() gin.HandlerFunc {
...
+		debtHash := c.Query("debtHash")
+
+		if len(debtHash) != txHashLength {
+			responseError(c, errParamInvalid, http.StatusBadRequest, apiParmaInvalid)
+			return
+		}
+		data, err := dbClinet.GetDebtByHash(debtHash)
+		if err == nil {
```
Comment (WTer) [inline api/handlers/blockinfo_handler.go]: no error case handler?

File: api/handlers/blockinfo_handler.go (hunk 1 above)
Comment (lizhengwei88) [reply api/handlers/blockinfo_handler.go]: Previous version dbClinet unity, all files have been replaced with dbClient

File: api/handlers/blockinfo_handler.go (hunk 2 above)
Comment (lizhengwei88) [reply api/handlers/blockinfo_handler.go]: ok

**Pull request merged**